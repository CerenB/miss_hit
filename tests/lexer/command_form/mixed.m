% (c) 2019 Florian Schanda
wibble a b, c            % the comma marks the end of the command
wibble a b, wibble b c   % two commands
wibble a b; c            % semicolon ends
wibble a b; wibble b c   % two commands
