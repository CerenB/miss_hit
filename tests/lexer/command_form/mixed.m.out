In mixed.m, line 1
| % (c) 2019 Florian Schanda
| ^^^^^^^^^^^^^^^^^^^^^^^^^^ info: [SL] COMMENT
In mixed.m, line 1
| % (c) 2019 Florian Schanda
|                           ^ info: NEWLINE
In mixed.m, line 2
| wibble a b, c            % the comma marks the end of the command
| ^^^^^^ info: [SL] IDENTIFIER
In mixed.m, line 2
| wibble a b, c            % the comma marks the end of the command
|        ^ info: CARRAY
In mixed.m, line 2
| wibble a b, c            % the comma marks the end of the command
|          ^ info: CARRAY
In mixed.m, line 2
| wibble a b, c            % the comma marks the end of the command
|           ^ info: COMMA
In mixed.m, line 2
| wibble a b, c            % the comma marks the end of the command
|             ^ info: [S ] IDENTIFIER
In mixed.m, line 2
| wibble a b, c            % the comma marks the end of the command
|                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ info: COMMENT
In mixed.m, line 2
| wibble a b, c            % the comma marks the end of the command
|                                                                  ^ info: NEWLINE
In mixed.m, line 3
| wibble a b, wibble b c   % two commands
| ^^^^^^ info: [SL] IDENTIFIER
In mixed.m, line 3
| wibble a b, wibble b c   % two commands
|        ^ info: CARRAY
In mixed.m, line 3
| wibble a b, wibble b c   % two commands
|          ^ info: CARRAY
In mixed.m, line 3
| wibble a b, wibble b c   % two commands
|           ^ info: COMMA
In mixed.m, line 3
| wibble a b, wibble b c   % two commands
|             ^^^^^^ info: [S ] IDENTIFIER
In mixed.m, line 3
| wibble a b, wibble b c   % two commands
|                    ^ info: CARRAY
In mixed.m, line 3
| wibble a b, wibble b c   % two commands
|                      ^ info: CARRAY
In mixed.m, line 3
| wibble a b, wibble b c   % two commands
|                          ^^^^^^^^^^^^^^ info: COMMENT
In mixed.m, line 3
| wibble a b, wibble b c   % two commands
|                                        ^ info: NEWLINE
In mixed.m, line 4
| wibble a b; c            % semicolon ends
| ^^^^^^ info: [SL] IDENTIFIER
In mixed.m, line 4
| wibble a b; c            % semicolon ends
|        ^ info: CARRAY
In mixed.m, line 4
| wibble a b; c            % semicolon ends
|          ^ info: CARRAY
In mixed.m, line 4
| wibble a b; c            % semicolon ends
|           ^ info: SEMICOLON
In mixed.m, line 4
| wibble a b; c            % semicolon ends
|             ^ info: [S ] IDENTIFIER
In mixed.m, line 4
| wibble a b; c            % semicolon ends
|                          ^^^^^^^^^^^^^^^^ info: COMMENT
In mixed.m, line 4
| wibble a b; c            % semicolon ends
|                                          ^ info: NEWLINE
In mixed.m, line 5
| wibble a b; wibble b c   % two commands
| ^^^^^^ info: [SL] IDENTIFIER
In mixed.m, line 5
| wibble a b; wibble b c   % two commands
|        ^ info: CARRAY
In mixed.m, line 5
| wibble a b; wibble b c   % two commands
|          ^ info: CARRAY
In mixed.m, line 5
| wibble a b; wibble b c   % two commands
|           ^ info: SEMICOLON
In mixed.m, line 5
| wibble a b; wibble b c   % two commands
|             ^^^^^^ info: [S ] IDENTIFIER
In mixed.m, line 5
| wibble a b; wibble b c   % two commands
|                    ^ info: CARRAY
In mixed.m, line 5
| wibble a b; wibble b c   % two commands
|                      ^ info: CARRAY
In mixed.m, line 5
| wibble a b; wibble b c   % two commands
|                          ^^^^^^^^^^^^^^ info: COMMENT
In mixed.m, line 5
| wibble a b; wibble b c   % two commands
|                                        ^ info: NEWLINE
mixed.m: lexed OK
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
